### ISSUE TITLE
Problems when generating Java Classes for Types that contain a fieldName "h"

### ISSUE THREAD

**Issue by KnyCeg** [apollographql/apollo-kotlin#4712]
### Version

3.7.4

### Summary

Generating Java code that contains a fieldName "h" leads to errors, cause the generated hashCode-method uses a variable "h", while fieldNames are not prefixed with "this.".

Simply prefixing fieldNames with "this." while generating the method [here](https://github.com/apollographql/apollo-kotlin/blob/main/libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/java/helpers/DataClass.kt#L174-L189) should resolve this issue.

### Steps to reproduce the behavior

Create a schema with a type like
```
type Test {
  h: String
}
```
and a query that uses this type.
Generate Java Code.

**Comment by martinbonnin** [comment]
Nice catch! fix is there: https://github.com/apollographql/apollo-kotlin/pull/4715

In the meantime, you can workaround by [using aliases](https://www.apollographql.com/docs/kotlin/advanced/using-aliases):

```graphql
query GetDimensions {
  dimensions {
    height: h
    width: w
  }
}
```

**Comment by BoD** [comment]
This has been released as part of [v3.7.5](https://github.com/apollographql/apollo-kotlin/releases/tag/v3.7.5)
